src/main/webapp/index.jsp

```diff
@@ -38,7 +38,10 @@
       style="width:75%; margin-left:auto; margin-right:auto; margin-top: 50px;">
 
       <h1>CodeU Chat App</h1>
-      <h2>Welcome!</h2>
+      <h2>Welcome to ~Team 31~'s chat app!</h2>
+      <p>Who is Team 31, you ask? We are an east coast-based team of CodeUers (plus
+          our PA Will, who is in Sunnyvale): Ellen Fu, Joyce Zhao, Pete Destil,
+          and Ransford Antwi! </p>
```

Comment:
nit: If including the termination tag `</p>`, I would prefer it on the next line and vertically aligned with the opening tag.